addons/google_calendar/models/calendar_recurrence_rule.py

```diff
@@ -15,6 +15,17 @@ class RecurrenceRule(models.Model):
     _name = 'calendar.recurrence'
     _inherit = ['calendar.recurrence', 'google.calendar.sync']
 
+    def _compute_rrule(self):
+        for recurrence in self:
+            current_rule = recurrence._rrule_serialize()
+            if recurrence.rrule != current_rule:
+                recurrence.write({'rrule': current_rule})
+
+    def _inverse_rrule(self):
+        for recurrence in self:
+            if recurrence.rrule:
+                values = self._rrule_parse(recurrence.rrule, recurrence.dtstart)
+                recurrence.write(dict(values))
```

Comment:
Indeed, super is not called. Normally, microsoft and google are not installed together but I get your point. I don't understand that you did not used the microsoft version if the microsoft module solve the issue.  Moreover, all these versions should have the same effect. In one case we use for rec in self.filtered which is not efficient and here we use an if which is better. microsoft adds need_sync_m=False but it should have no effect on google records. :thinking:

Reply:
Could you add a comment explaining why this is needed? I fear that when a cleaup is done, people will just remove the override and delete it without a blink. 
I think that in the end, it is still strange because depending on the module installation order, the Microsoft version will be installed or the Google one and it will be random. 
Unfortunately, we can't easily distinguish both cases without horrible checks.